test/metabase/util_test.cljc

```diff
@@ -470,11 +472,20 @@
     (is (= [:c :e] (u/conflicting-keys {:a 1 :b 2 :c 3 :e nil}
                                        {:b 2 :c 4 :d 5 :e 6})))))
 
+(defn- join [& crumbs] (str/join ":" crumbs))
```

Comment:
As the idiot who wrote this original test, I probably should have just used `vector`:

```clojure
(deftest map-all-test
  (testing "map-all with 1 collection is just map"
    (is (= [[0] [1] [2] [3] [4]] (u/map-all vector (range 5)))))
  (testing "map-all works with 3 collections"
    (is (= [[0 0] [1 1] [2 2] [3 nil] [4 nil]] (u/map-all vector (range 5) (range 3)))))
  (testing "map-all works with higher arity"
    (is (= [[0 0 0] [1 1 1] [2 2 2] [3 nil 3] [nil nil 4]] (u/map-all vector (range 4) (range 3) (range 5))))))
```

Reply:
Since you don't use the the result of the function in any way in the property-based test, I'm not sure it's worth hoisting the function out even, use whatever n-arity built-in you want.